airflow/config_templates/config.yml

```diff
@@ -658,6 +658,8 @@ logging:
       description: |
         Whether the local log files for GCS, S3, WASB and OSS remote logging should be deleted after
         they are uploaded to the remote location.
+        If you are deploying Airflow with Helm Chart this setting won't work. Instead, you should
+        enable ``logGroomerSidecar``.
```

Comment:
I also got it when using Elasticsearch remote logging. We need to be sure this happens also in the other technologies. Maybe @hussein-awala who wrote this feature https://github.com/apache/airflow/pull/29772 can help us to be sure.

Reply:
@hussein-awala any thoughts on this?

Reply:
IMHO, it should work with helm chart deployments too, I will take a look.

> I also got it when using Elasticsearch remote logging.

You faced the issue with helm?

Reply:
yes when deployed with helm the `delete_local_logs` flag had no effect.